src/components/presentational/Histogram/Histogram.stories.tsx

```diff
@@ -1,38 +1,47 @@
-﻿import React from "react";
-import Histogram, {HistogramProps} from "./Histogram";
-import Layout from "../Layout";
-import { Meta, StoryObj } from "@storybook/react";
+﻿import React from 'react';
+import { Layout } from '../../presentational';
+import Histogram, { HistogramProps } from './Histogram';
 
-const meta: Meta<typeof Histogram> = {
-	title: "Presentational/Histogram",
-	component: Histogram,
-	parameters: {
-		layout: 'fullscreen'
-	}
+export default {
+    title: 'Presentational/Histogram',
+    component: Histogram,
+    parameters: { layout: 'fullscreen' }
 };
 
-export default meta;
-type Story = StoryObj<typeof Histogram>;
+const render = (args: HistogramProps) => <Layout colorScheme="auto"><Histogram {...args} /></Layout>
```

Comment:
You can define this render function on the exported metadata above so you don't have to include it on each of the story definitions.

```
export default {
    title: 'Presentational/Histogram',
    component: Histogram,
    parameters: { layout: 'fullscreen' },
    render: (args: HistogramProps) => <Layout colorScheme="auto"><Histogram {...args} /></Layout>
};
```

Also, if you revert to the `const meta: Meta<typeof Histogram> = {` version of this, you get type checking on the render function signature, etc.